ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1686,6 +1687,39 @@ daemons:
 
 This will add a dameon to the eos configuration that is most useful when trying to run OpenConfig clients like ocprometheus
 
+#### Connectivity Monitor
+
+```yaml
+monitor_connectivity:
+    shutdown: < true | false >
```

Comment:
inconsistent indentation. Should be two spaces for yaml.

Reply:
Thank you. I will fix this